tests/unit/C/README.rst

```diff
@@ -19,6 +22,12 @@ Unfortunately this does not install the libraries and a manual build set is requ
 - sudo cmake --build build
 - sudo cp build/libgtest* /usr/lib
 
+Red Hat Enterprise Linux:
+-------------------------
+
+- sudo yum install gtest gtest-devel
```

Comment:
We need to add the following line before installing gtest:
```
sudo yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```

Reply:
Updated.